package.json

```
@@ -32,7 +32,8 @@
     "test": "node Makefile.js test",
     "test:cli": "mocha",
     "test:fuzz": "node Makefile.js fuzz",
-    "test:performance": "node Makefile.js perf"
+    "test:performance": "node Makefile.js perf",
+    "test:wdio": "node Makefile.js wdio"
```

wido is just the used tool, and the gha job name is `test-on-broswer`. :)
```suggestion
    "test:browser": "node Makefile.js wdio"
```

---

Done in 9b57f7efa3d4bd687f29eb1e767025e26e681cf0, thanks.